Resources/Computing/PoolComputingElement.py

```diff
@@ -26,6 +26,11 @@
 
 def executeJob(executableFile, proxy, taskID, **kwargs):
   """ wrapper around ce.submitJob: decides which CE to use (Sudo or InProcess)
+
```

Comment:
You are missing the docstring for `executableFile`:

```suggestion
:param str executableFile: location of the executable file
```